### PR TITLE
Add utility classes for fade-in animation

### PR DESCRIPTION
**Pull request by Malinicz**
Add 2 classes for fade-in animation, widely used in brainly-frontend app.

### CODE REVIEW

**Review by matzimowski — APPROVED** [review]
lgtm

File: src/sass/_utils.scss [hunk 1]
```diff
@@ -31,3 +31,25 @@
...
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.sg-animation-fade-in-slow {
```
Comment (matzimowski) [inline src/sass/_utils.scss:45]: what about fadeOut? fadeIn gonna miss it 🤗

File: src/sass/_utils.scss [hunk 2]
```diff
@@ -31,3 +31,25 @@
...
+  animation-name: sgAnimationFadeIn;
+  animation-duration: 600ms;
+  animation-timing-function: ease-in;
+}
+
+.sg-animation-fade-in-fast {
+  animation-name: sgAnimationFadeIn;
+  animation-duration: 400ms;
+  animation-timing-function: ease-in;
```
Comment (coderitual) [inline src/sass/_utils.scss:54]: I found only 3 usages of this particular easing in application which is kind of wrong. Animation principles sugges easing out or in out like for things we're showing, can we please discuss? :)

File: src/sass/_utils.scss (hunk 1 above)
Comment (coderitual) [reply src/sass/_utils.scss:45]: @matzimowski that's true :) please keep in mind this is kind of temporary. To cover fade out properly we need transition component taking care of keeping instance until animation ends. I hope we get there when we will have motion designer :)

File: src/sass/_utils.scss (hunk 2 above)
Comment (coderitual) [reply src/sass/_utils.scss:54]: ah probably you meant usage of these: 
image

**Review by coderitual — APPROVED** [review]

**Review by bartoszlorek — APPROVED** [review]

**Pull request merged**